third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -176,12 +176,9 @@ const TEST_CONFIGS: Lazy<BTreeMap<&str, TestConfig>> = Lazy::new(|| {
             exp_suffix: None,
             options: opts
                 .clone()
-                .set_experiment(Experiment::AST_SIMPLIFY_FULL, true)
-                // TODO: this check should not need to be turned off, but some
```

Comment:
It was not clear what about the tests were needed to be fixed - they seemed fine to me (enabling it allows various errors to be reported in the stackless bytecode checker, which seems like useful tests).